src/components/subHeader/components/description/Description.js

```diff
@@ -0,0 +1,48 @@
+// Libraries
+import PropTypes from 'prop-types'
+
+// components
+import PlaceHolderList from "../placeHolderList";
+import DescriptionHeading from "../descriptionHeading";
+
+// css
+import './description.css'
+
+
+function Description(props){
+
+    const { restaurantInfo, additionalInfo } = props;
+
```

Comment:
Try to use renderProps

Reply:
i can use it but i am not reusing that component anywhere else and using render prop will make the description component much more complex. i have previously used but removed after words